website/docs/docs/build/packages.md

```diff
@@ -22,13 +22,14 @@ dbt _packages_ are in fact standalone dbt projects, with models and macros that
 * Models in the package will be materialized when you `dbt run`.
 * You can use `ref` in your own models to refer to models from the package.
 * You can use macros in the package in your own project.
+* It's important to note that defining and installing dbt packages is different from [defining and installing Python packages](/docs/build/python-models#using-pypi-packages)
 
-:::note Using Python packages
-
-Defining and installing dbt packages is different from [defining and installing Python packages](/docs/build/python-models#using-pypi-packages).
 
+:::info Project Dependencies versus Packages
+ Packages are different to Project dependencies, a feature that allows cross-project `ref`. Refer to [Project dependencies](/docs/collaborate/govern/project-dependencies) for more info about the difference between `dependencies.yml` and `packages.yml`.
```

Comment:
> Refer to [Project dependencies](/docs/collaborate/govern/project-dependencies) for more info about the difference between `dependencies.yml` and `packages.yml`.

This doesn't sound quite right to me.

Points we should make clear here:
- `dependencies.yml` is the successor to `packages.yml`. It can include both types of dependencies: `packages` and `projects`.
- "Package" dependencies are a way of adding the source code from someone else's dbt project into your own, like a library. "Project" dependencies are a different way of building on top of someone else's work in dbt. For more information, see <other page>
- You can rename `packages.yml` &rarr; `dependencies.yml` — UNLESS you need to use Jinja within your `packages` specification, such as to stick an environment variable with a git token in a private `git` package specification.